src/features/calendar/components/index.tsx

```diff
@@ -13,12 +14,62 @@ export enum TimeScale {
   MONTH = 'month',
 }
 
+function isValidDate(date: Date | unknown) {
+  return date instanceof Date && !isNaN(date.getTime());
+}
```

Comment:
This exists nicely in `dayjs(date).isValid()`

Reply:
Of course is does, I feel silly not having checked that from the start :)
I pushed a new commit that uses dayjs instead.